util/types/json/functions.go

```diff
@@ -124,6 +129,23 @@ func unquoteString(s string) (string, error) {
 	return ret.String(), nil
 }
 
+// decodeEscapedUnicode decodes unicode into utf8 bytes. e.g.
+// decodeEscapedUnicode("4321") -> "䌡"
+// decodeEscapedUnicode("0034") -> "4"
+func decodeEscapedUnicode(s []byte) (char []byte, err error) {
+	var buffer [6]byte
```

Comment:
`6` and `2` are hard to understand.
may implement as the demo [here](https://golang.org/pkg/encoding/hex/#Decode) will be easier to understand?